omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/DropDownFragmentController.java

```diff
@@ -24,10 +28,17 @@ public void controller(FragmentConfiguration config) throws Exception {
             	Map < String, Object > option = new HashMap < String, Object > ();
                 option = new HashMap < String, Object > ();
                 option.put("value", concept.getId().toString());
-                option.put("label", concept.getName(Context.getUserContext().getLocale()).getName());
+                option.put(OPTION_LABEL, concept.getName(Context.getUserContext().getLocale()).getName());
                 options.add(option);
             }
+            Collections.sort(options, mapComparator);
```

Comment:
Since it's very much a one-off thing, you may just inline it:
```java
Collections.sort(options, new Comparator<Map<String, Object>>() {
    public int compare(Map<String, Object> m1, Map<String, Object> m2) {
        return ((String) m1.get(OPTION_LABEL)).compareTo((String) m2.get(OPTION_LABEL));
    }
});
```

Reply:
Cool, thanks @mks-d. This is addressed.